ecommerce/views.py

```diff
@@ -57,6 +59,16 @@ class ProductViewSet(ModelViewSet):
     queryset = Product.objects.all()
 
 
+class CompanyViewSet(ModelViewSet):
+    """API view set for Companies"""
+
+    permission_classes = (IsAuthenticated,)
```

Comment:
Will change to `ReadOnlyModelViewSet`, but permissions were discussed with Ferdi and Peter, there is a possibility that a list of companies might need to be presented to users so that's why just `IsAuthenticated` was used.

Reply:
That's fair, but it should be `(IsAuthenticatedOrReadOnly | IsAdminUser,)` instead then, since I don't think allowing any user to create or update companies except admin users.

Reply:
`ReadOnlyModelViewSet` will prevent creates/updates, at this point it's only for display purposes and companies can be created in django admin